### ISSUE TITLE
Implement a logout flow triggered from within the hub

### ISSUE THREAD

**Issue by jits** [appvia/appvia-hub#67]
Needs some way to call out to the auth proxy to perform the logout (may be as easy as using the right URL that the auth proxy then handles instead of the upstream hub server).

**Comment by jits** [comment]
Reopened so that:
a) we can make the env var (and thus the logout feature) optional
b) look into getting it working with the current demo hubs set up (/cc @davet1985 and @tasharnvb)

**Comment by davet1985** [comment]
After a short investigation, I've found the logout path for the OAuth proxy we are using (https://pusher.github.io/oauth2_proxy/). It's `/oauth2/sign_out`.

I've tested this and it works. After logout, the user will land on the GitHub login page to reauthenticate. However, if there is another GitHub session found then the user will be signed out, redirected to the root path and automatically go through the login flow once again, landing on the hub home page. To some, this may not have appeared to have signed out.

I'll configure any demos to use this.

@jits why would we want to make the logout feature optional? I can't really think of a legitimate use case for not needing it? If we don't, then we could consider this issue done?
